client/state/reader/posts/normalization-rules.js

```diff
@@ -62,12 +62,13 @@ const hasShortContent = post => getCharacterCount( post ) <= 100;
  */
 export function classifyPost( post ) {
 	const canonicalImage = post.canonical_image;
+	const imagesForGallery = post.content_images && filter( post.content_images, imageIsBigEnoughForGallery );
```

Comment:
[filter](https://github.com/lodash/lodash/blob/4.17.4/lodash.js#L9188) is quite safe and will return an empty array if given a bad collection.

No need to guard with `post.content_images && `